tests/integration/test_vulnerability_detector/test_feeds/msu/test_missing_fields_msu_feed.py

```diff
@@ -75,15 +128,87 @@ def remove_field_feed(request):
+            brief: Clean vulnerabilities tables.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+
+    assertions:
+        - Verify that the number of vulnerabilities is equal to the expected number of vulnerabilities inserted in the
+          VULNERABILITIES table of CVE DB.
+
+    input_description:
+        - Test cases are defined in the custom_msu.json file. This file contains different kinds of vulnerabilities.
+          This test gets their configuration of the wazuh_invalid_msu_feed.yaml file.
+
+    expected_output:
+        - r'INFO: .* The update of the .* feed finished successfully.'
+
+    tags:
+        - vulnerability
+        - vulnerability_detector
+        - system_inventory
+    '''
     vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.MSU_LOG,
                                         expected_vulnerabilities_number=0)
 
 
 def test_invalid_msu_feed(clean_vuln_tables, get_configuration, configure_environment, remove_field_feed):
-    """
-    Check if vulnerability detector behaves as expected when importing MSU feed with missing fields
-    """
+    '''
+    description: Test to check vulnerability detector behavior when importing the MSU feed with missing tags. To do
+                 this, a tag is removed from the feed and check if the vulnerability can be inserted into the database.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean vulnerabilities tables.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - remove_tag_feed:
+            type: fixture
+            brief: Modify the feed by removing a certain field and loading the new configuration.
+
+    assertions:
+        - Verify that the number of vulnerabilities is equal to the expected number of vulnerabilities inserted in the
+          VULNERABILITIES table of CVE DB and if the process is running.
+
+    input_description:
+        - Test cases are defined in the custom_alas_feed.json and custom_alas2_feed.json files. Those files contain two
+          different kinds of vulnerabilities. 'fields' is the list of tags to be checked and 'field_ids' is the
+          dictionary's list that indicates the missing tag. This test gets their configuration of the
+          wazuh_invalid_msu_feed.yaml file.
+
+    expected_output:
+        - r'ERROR: The .* feed couldn't be parsed from .* file'
+        - r'ERROR: CVE database could not be updated.'
```

Comment:
```suggestion
        - r'INFO: .* The update of the .* feed finished successfully.'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/bf3acae464afb8efc7395d17a683e3aa9c84be9c)